.env.development

```diff
@@ -20,3 +20,4 @@ ACCOUNT_PROFILE_URL='http://localhost:1995/u'
 MFE_CONFIG_API_URL='http://localhost:18000/api/mfe_config/v1'
 INSTITUTION_PORTAL_PATH=''
 COURSE_OPERATIONS_API_V2_BASE_URL='http://localhost:18000/pearson_course_operation/api/v2'
+COURSE_OPERATIONS_API_METRICS_BASE_URL='http://localhost:18000/pearson_course_operation/api/metrics'
```

Comment:
I have one question here, 
I see that in many tests you are using for example 'http://localhost:18000/pearson_course_operation/api/v2', why not add the variables `COURSE_OPERATIONS_API_V2_BASE_URL` and `COURSE_OPERATIONS_API_METRICS_BASE_URL` in _.env.test_. In addition to adding them in .env so that they are empty so that there is a correspondence. @sergivalero20 FYI @AuraAlba